### ISSUE TITLE
GPU is not detected on lambda labs servers while creating a build

### ISSUE THREAD

**Issue by aroop** [kiri-art/docker-diffusers-api-runpod#2]
I do not have access to GPU locally, so I create a docker build on lambda labs servers. 

While running the build command in the readme I see the following messages through build seems to work

```
#7 3.730 [device] CPU only - no GPU detected
#7 3.730 MPS not available because the current PyTorch install was not built with MPS enabled.
```

It could be realted to https://github.com/kiri-art/docker-diffusers-api/commit/682a34221f5b586fd0d8e9c0789201cb238cf225

**Comment by gadicc** [comment]
Yeah, this is just for informational purposes.  We don't need any GPU during build time, so you can safely ignore.  Let's leave this open though as maybe I'll add a comment about this somewhere.  Thanks for reporting.
